### PR TITLE
ISSUE-50 Alexander Bashkirov

### PR DESCRIPTION
**Pull request by bash-spbu**
Uploaded solutions & one task changed

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -18,9 +18,9 @@
 54 - 76
 72 - 38
 97 + 71
-31 - 67
-81 + 44
-47 - 71
+31 - 67 = 0
```
Comment (Ayokontol) [inline tasks]: Wrong answer, please, fix it.

Comment (bash-spbu) [reply tasks]: Oops, thank you!

**Comment by bash-spbu** [comment]
The force push above was to update commit message, I ocassionaly forgot it.

**Comment by bash-spbu** [comment]
The mistake is fixed now.

**Pull request merged**